cdap-ui/cypress/integration/pipeline.complexschema.spec.ts

```diff
@@ -206,6 +206,9 @@ describe('Output Schema', () => {
       cy.get('[data-testid="deploy-pipeline"]').click();
       cy.url({ timeout: 60000 }).should('include', `/view/${PIPELINE_NAME}`);
 
+      // Wait to allow pipeline to finish rendering before checking node properties
+      cy.wait(500);
```

Comment:
Is 500ms enough? What happens on a large pipeline? Is rearranging the nodes a constant time?

Reply:
I suspect rearranging the nodes is not constant time, and that it would take longer for larger pipelines with more nodes and connections.